manifests/init.pp

```diff
@@ -12,9 +12,12 @@
   Optional[Sensitive] $host_factory_token = $conjur::params::host_factory_token,
 ) inherits conjur::params {
   if $cert_file {
-    $ssl_certificate = file($cert_file)
+    $raw_ssl_certificate = file($cert_file)
+  } else {
+    $raw_ssl_certificate = $ssl_certificate
   }
```

Comment:
I wonder if the scope of the changes can be reduced by using a different var in L4 instead and setting `$ssl_certificate` directly here?
```puppet
...
Optional[String] $raw_ssl_certificate = $conjur::params::ssl_certificate,
...
  if $cert_file {
    $ssl_certificate = file($cert_file)
  } else {
    $ssl_certificate = $raw_ssl_certificate
  }
...
$client = conjur::client($appliance_url, $version, $ssl_certificate)
```

This should make the variable name used in other `.pp` files the same and more intuitive than having mismatching param name (`params::ssl_certificate`) to the resulting variable (`raw_ssl_certificate`).

Thoughts?

Reply:
@sgnn7 I thought to do that but it would make the upcoming release non-backwards compatible since the conjur class would no longer take the `ssl_certificate` param, instead it would take `raw_ssl_certificate`.